bazel/BUILD

```diff
@@ -7,6 +7,30 @@ exports_files([
     "sh_test_wrapper.sh",
 ])
 
+genrule(
+    name = "gnu_build_id",
+    outs = ["gnu_build_id.ldscript"],
+    cmd = """
+      echo --build-id=0x$$(
+          grep STABLE_BUILD_SCM_REVISION bazel-out/stable-status.txt \\
+        | sed 's/^STABLE_BUILD_SCM_REVISION //') \\
+        > $@
+    """,
+    stamp = 1,
```

Comment:
What is `stamp` doing here and in the other genrules? I can't see much in the way of docs for genrule+stamp?

Reply:
`stamp=1` gives the rule access to the `bazel-out/stable-status.txt` and `bazel-out/volatile-status.txt` files. It also makes the genrule depend directly on the stable status, so it'll re-run if the status changes.

The param is undocumented :( : https://github.com/bazelbuild/bazel/issues/4942